### DISCUSSION TITLE
Issues with outputplot

### DISCUSSION THREAD

**Discussion by KlPGSL** [pastas/pastas#677] [Q&A] [answered]
Hello everyone,

is discovered this fantastic tool a while ago. 
While testing it a little bit I encountered the following issue: I provided a timeline of a river where the head varies between 70 and 75 meters. In the outputplot the head is somehow shifted downwards around 50 meters and squeezed. Whats the cause of this? Is the intentional?

Secound issue is that the output timeline of the recharge is somehow normed?! How can I display the real values in the output plot?

![Figure_1](https://github.com/pastas/pastas/assets/157016785/2508cf49-6b67-4e00-b4cc-4a2a76ede22f)

```
# Model settings
tmin = pd.Timestamp("2003-01-01") 
tmax = pd.Timestamp("2022-12-31")   

noise = True

ml = ps.Model(head)
rch = ps.rch.FlexModel(gw_uptake=False)

rm = ps.RechargeModel(rain, evap, recharge=rch, rfunc=ps.Gamma(), name="rch")
ml.add_stressmodel(rm)

ml.set_parameter("rch_srmax", vary=True)
ml.set_parameter("rch_kv", vary=True)
ml.set_parameter("rch_lp", vary=True, initial=0.25)
ml.set_parameter("constant_d", vary=True, initial=70.25, pmax=head.min())   #Start-Head anpassen

ml.add_noisemodel(ps.ArmaModel())

w = ps.StressModel(riv, rfunc=ps.Exponential(), name="riv", settings="waterlevel")
ml.add_stressmodel(w)

solver = ps.LmfitSolve()
ml.solve(
        tmin=tmin,
        tmax=tmax,
        noise=True,
        solver=solver,
        method="least_squares",
        report="basic",
    )
ml.plots.results(figsize=(12, 8))
plt.show()
```

**Comment by rubencalje** [comment, marked as answer]
Hi @KlPGSL,

The unit of the two plots on the bottom left are the same as the head-series: it is the influence of recharge and the river level on the head. Variations in river-level result in changes in head of 29 % of the river-variations (the value of parameter riv_A). That is why the absolute values of the lower left plot are about 1/3th of the absolute river value.  The absolute value of these plots are not so interesting, and depend on the way you preprocessed your input-series (if you normalized them or not for example). What is important in these plots is the variation.

So the plot does not show the value of the groundwater recharge. There is a way to get this recharge from the flexmodel I think, but maybe one of the other developeres can help you with that.

**Reply by dbrakenhoff** [reply]
Hi @KlPGSL, this notebook shows how you can estimate the recharge from the recharge model:

https://pastas.readthedocs.io/en/latest/examples/recharge_estimation.html

**Reply by KlPGSL** [reply]
Thanks for the clarification. I totally missunderstood the output.
